core/src/main/java/dk/alexandra/fresco/lib/math/integer/exp/ExponentiationOpenExponent.java

```diff
@@ -46,12 +46,11 @@ public ExponentiationOpenExponent(Computation<SInt> x, BigInteger e) {
   @Override
   public Computation<SInt> build(SequentialNumericBuilder builder) {
     if (exponent.equals(BigInteger.ZERO)) {
-      return builder.numeric().known(BigInteger.valueOf(1));
+      throw new RuntimeException("Now why would you exponentiate a SInt with 0?");
```

Comment:
Agree, branch updated.

Given that this is the open exponentiate, you will now - however maybe not runtime.